lib/Screens/splash.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:pispapp/theme/light_color.dart';
+
+class SplashScreen extends StatelessWidget {
+  Widget _getStartedTile(BuildContext context) {
```

Comment:
is this something reusable? if yes perhaps can put it under `lib/widgets`? otherwise i would like to just suggest to rename the function name to `_buildGetStartedTile`